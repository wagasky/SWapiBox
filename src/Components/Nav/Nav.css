.nav, 
ul { 
  padding: 0px 0px;
  margin: 0;
  background: black;
}

li {
  padding: 0;
  margin: 0;
}

ul .nav {
  flex-direction: column;
  align-content: center;
  height: 200px;
}

a .nav {
  text-decoration: none;
  background-color: black;
  color: white;
}

nav ul button .nav {
  text-decoration: none;
  width: 200px;
  background-color: black;
  color: white;
}
