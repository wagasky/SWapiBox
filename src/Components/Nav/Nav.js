import React from 'react';
import { NavLink } from 'react-router-dom';
import { func } from 'prop-types';


const Nav = ({ handleButton }) => {

  return (
    <div>
      <nav>
        <ul>
          <button><NavLink to='/people'>People</NavLink></button>
          <button onClick={() => handleButton('vehicles')}><NavLink to='/vehicles'>Vehicles</NavLink></button>
          <button onClick={() => handleButton('planets')}><NavLink to='/planets'>Planets</NavLink></button>
          <button><NavLink to='/favorites'>Favorites</NavLink></button>
        </ul>
      </nav>
    </div>
  );
};

Nav.propTypes = {
  handleButton: func,
};

export default Nav;