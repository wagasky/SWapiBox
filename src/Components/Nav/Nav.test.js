/* eslint-disable */ 

import React from 'react';
import ReactDOM from 'react-dom';
import Nav from './Nav';
import { shallow } from 'enzyme';

it.skip('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Nav />, div);
  ReactDOM.unmountComponentAtNode(div);
});