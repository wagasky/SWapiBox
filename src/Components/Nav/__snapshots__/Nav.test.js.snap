// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav should match snapshot 1`] = `
<div>
  <nav>
    <ul>
      <button>
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          to="/people"
        >
          People
        </NavLink>
      </button>
      <button
        onClick={[Function]}
      >
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          to="/vehicles"
        >
          Vehicles
        </NavLink>
      </button>
      <button
        onClick={[Function]}
      >
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          to="/planets"
        >
          Planets
        </NavLink>
      </button>
      <button>
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          to="/favorites"
        >
          Favorites
        </NavLink>
      </button>
    </ul>
  </nav>
</div>
`;
